### PR TITLE
Adds the b english element

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_apple:  Ready to go!  Needs thumbs please!  Thanks!

### CODE REVIEW

File: elements/b.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Originally, &lt;b&gt; was meant to make text boldface.  Currently, it is used to indicate a section of text that is styled differently from other text.  The &lt;b&gt; element is usually used when other command codes don't work well for the situation.  It is also used to indicate keywords, product names, and other text that need to stand out but do not have any emphasis on the text.  It is strongly recommended to use &lt;em&gt;, &lt;strong&gt; or &lt;mark&gt; when indicating important text.</p>
```
Comment (ericanaughton) [inline elements/b.html]: I am concerned that you strongly recommend using `<mark>` over `<b>`. I did a quick search on popular HTML tags, and `<mark>` is not found in any of these resources:

http://www.templatesheaven.com/blog/23-most-popular-html-tags-you-are-likely-to-see-in-templates/
http://www.dummies.com/how-to/content/common-html-tags.html
http://www.simplehtmlguide.com/cheatsheet.php
http://www.99lime.com/_bak/topics/you-only-need-10-tags/

Comment (kathyoleary) [inline elements/b.html]: Does that work better?  I want to mention the others so people know what options there are.

**Comment by werbeckes** [comment]
@kathyoleary if this is the "old school" way of doing things why are we adding it to LMHT?  I thought we were trying to focus on the most important tags that a newbie needs to know.

**Comment by whazzmaster** [comment]
:+1:

**Comment by bigtiger** [comment]
:+1:

**Pull request merged**